onnx/defs/data_type_utils.cc

```diff
@@ -391,8 +391,8 @@ TypesWrapper::TypesWrapper() {
   type_str_to_tensor_data_type_["uint16"] = TensorProto_DataType_UINT16;
   type_str_to_tensor_data_type_["uint32"] = TensorProto_DataType_UINT32;
   type_str_to_tensor_data_type_["uint64"] = TensorProto_DataType_UINT64;
-  type_str_to_tensor_data_type_["complext64"] = TensorProto_DataType_COMPLEX64;
-  type_str_to_tensor_data_type_["complext128"] =
+  type_str_to_tensor_data_type_["complex64"] = TensorProto_DataType_COMPLEX64;
```

Comment:
lol I'm wondering whether there is typo checker we can add to the CI :-)

Reply:
Fair point, will add some test cases.